packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -145,6 +145,24 @@ bool debugProfileLayoutsEnabled = false;
 ///    areas are being excessively repainted.
 bool debugProfilePaintsEnabled = false;
 
+/// Adds debugging information to [Timeline] events related to [RenderObject]
+/// layouts.
+///
+/// This flag will only add [Timeline] event arguments for debug builds.
+/// Additional arguments will be added for the "LAYOUT" timeline event and for
```

Comment:
same here: you get additional stats on dirtyness and the RenderObject events contain their diagnostic information.

Reply:
done